catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -369,6 +471,9 @@ private Metacard readMetacard(File file) throws IngestException {
             }
         }
 
+        if (!ignoreContent) {
+            createContentItem(result);
```

Comment:
If you are ingesting content then you need to send a CreateStorageRequest to the catalog framework.